### PR TITLE
Adding missing pytest dependency

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #7753

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	# TODO: pbindings won't work here. need to recreate this functionality
+	pbindings pulpcore python
+
+test-unit: setup-virtualenv
+	$(VENV)/bin/django-admin test --noinput
+
+test-functional: setup-virtualenv bindings
+	$(VENV)/bin/pytest -v -r sx --color=yes --pyargs pulpcore.tests.functional
+
+test: test-unit test-functional
```
Comment (mdellweg) [inline Makefile:40]: This is not, what the commit message promises.

**Review by mdellweg — COMMENTED** [review]
I think the .venv parts are unrelated, too.

**Comment by daviddavis** [comment]
> I think the .venv parts are unrelated, too.

How about a separate commit then?

**Comment by mdellweg** [comment]
> > I think the .venv parts are unrelated, too.
> 
> How about a separate commit then?

I thought the .venv things were related to that makefile you were drafting.

**Comment by daviddavis** [comment]
No, not necessarily. I am using .venv to store my dependencies when I work on pulpcore outside of our vagrant environment. It's a common convention to ignore `.venv`. See [black's config](https://github.com/psf/black/blob/989ea69bd1d0a02aa6c5794d941c122f7b4958b4/pyproject.toml#L19) as an example.

**Comment by daviddavis** [comment]
Here's another example: https://github.com/github/gitignore/blob/master/Python.gitignore#L109

**Review by mdellweg — APPROVED** [review]

**Review by goosemania — APPROVED** [review]

**Pull request merged**